rs/ethereum/cketh/minter/src/eth_logs/mod.rs

```diff
@@ -260,211 +232,6 @@ pub enum EventSourceError {
     InvalidEvent(String),
 }
 
-impl TryFrom<LogEntry> for ReceivedEvent {
```

Comment:
The problem with this approach is that it parses all events in a single funnel, so that even if we know we are parsing event form log scraping 1, we would still accept events from log scraping 2. Having it as separate implementations of a trait (`LogParser`) solves that problem.